### PR TITLE
Avoid generating an empty setenv =

### PR DESCRIPTION
**Pull request by mgedmin**
Because it breaks packages that have a

    test-additional = ["setenv =", ...]

in their .meta.toml, like zope.publisher.

### CODE REVIEW

**Comment by dataflake** [comment]
Why wouldn't you take those `setenv` lines out of `test-additional` and add them to `testenv-setenv`?

**Comment by mgedmin** [comment]
> Why wouldn't you take those setenv lines out of test-additional and add them to testenv-setenv?

Oh, I will.

But there's no reason to generate empty `setenv =` here anyway, is there?

**Comment by dataflake** [comment]
There is no reason to generate an empty one, that is correct. I'm still worried that other things will break. Some of these replacements that get handed from template to sub-template are hard to follow. Go ahead and we'll see what happens.

**Review by dataflake — APPROVED** [review]
